cmd/up/activate.go

```diff
@@ -243,9 +246,11 @@ func (up *upContext) shouldRetry(ctx context.Context, err error) bool {
 }
 
 func (up *upContext) devMode(ctx context.Context, app apps.App, create bool) error {
```

Comment:
I'd also time the wait in `DevContainerCreation`. I don't know if it should be another prop because it depends on the connection with the cluster

Reply:
this is trakced with `devContainerCreationDurationSeconds ` maybe i added the count in a missplace?

Reply:
Oh, I see it now. Sorry for the noise